libecl/include/ert/ecl/ecl_nnc_export.h

```diff
@@ -38,6 +38,8 @@ typedef struct {
   int grid_nr2;
   int global_index2;
 
+  int nnc_index;
```

Comment:
Mabye add a comment here: "The `ncc_index` corresponds to the input ordering of this nnc."